crates/sui-json-rpc/src/transaction_execution_api.rs

```diff
@@ -217,6 +218,7 @@ impl WriteApiServer for TransactionExecutionApi {
     ) -> RpcResult<SuiTransactionBlockResponse> {
         Ok(self
             .execute_transaction_block(tx_bytes, signatures, opts, request_type)
+            .instrument(error_span!("execute_transaction_block"))
```

Comment:
isn't this covered by the regular span on execution?

Reply:
I believe that span applies to both validators and fullnodes, while this span is exclusive to fullnodes, because it's in the implementation of the write API.